pynamodb/connection/table.py

```diff
@@ -304,6 +304,7 @@ def create_table(
         local_secondary_indexes: Optional[Any] = None,
         stream_specification: Optional[Dict] = None,
         billing_mode: str = DEFAULT_BILLING_MODE,
+        tags: Optional[Dict] = None,
```

Comment:
Per https://docs.aws.amazon.com/amazondynamodb/latest/developerguide/TaggingRestrictions.html:
```suggestion
        tags: Optional[Dict[str, str]] = None,
```

Reply:
Fixed